examples/dpsgd_example.py

```diff
@@ -0,0 +1,185 @@
+# Copyright 2021 DeepMind Technologies Limited. All Rights Reserved.
```

Comment:
Is this copyright header ok? A big portion of the code is from the JAX repo.

Reply:
Yes this is fine, but please add an attribution in the file doc-string explaining that a big portion of the code is forked from the JAX example

Reply:
Done!